.changeset/five-dolphins-relax.md

```diff
@@ -0,0 +1,8 @@
+---
+"@justeattakeaway/pie-icons-react": patch
+"@justeattakeaway/pie-icons-vue": patch
+"@justeattakeaway/pie-icons": patch
```

Comment:
I'm not entirely sure we need to release a patch for all icons, since it doesn't change anything content wise.

wdyt?

Reply:
Yeah, if you're not updating the version used in the vue and react packages then I don't see why those two packages would have a version bump.

Reply:
Yeah, feel free to remove the vue/react changes. Changesets will auto-bump the pie-icons dep once this PR gets versioned :)

Reply:
Thank you peeps!

Reply:
Good question, @siggerzz surely knows that better than I do.

Reply:
Faye has also done a change on the icons and it auto added all the packages, so I think this is intended Changeset behaviour to bump all three packages in tandem. So if you can add it back in happy to approve it then